wd_helper.au3

```diff
@@ -696,14 +696,34 @@ EndFunc   ;==>_WD_HighlightElement
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_HighlightElements($sSession, $aElements, $iMethod = Default)
+	Local Const $sFuncName = "_WD_HighlightElements"
+	Local Const $aMethod[] = _
+			[ _
+			"border: 0px", _
+			"border: 2px dotted red", _
+			"background: #FFFF66; border-radius: 5px; padding-left: 3px;", _
+			"border: 2px dotted red; background: #FFFF66; border-radius: 5px; padding-left: 3px;" _
```

Comment:
The first two styles doesn't include the trailing semicolon `;`.
The others do include the trailing semicolon `;`.

I believe it doesn't matter, but I would go with the semicolon for all styles, even when it's just one style.
Same for the function above _WD_HighlightElement.